pkg/logs/config/config.go

```diff
@@ -176,51 +176,86 @@ func buildTCPEndpoints() (*Endpoints, error) {
 	return NewEndpoints(main, additionals, useProto, false, 0), nil
 }
 
+// LogsConfigKeys stores logs configuration keys stored in YAML configuration files
+type LogsConfigKeys struct {
+	UseCompression          string
+	CompressionLevel        string
+	ConnectionResetInterval string
+	LogsDDURL               string
+	LogsNoSSL               string
+	DDURL                   string
+	DevModeNoSSL            string
+	AdditionalEndpoints     string
```

Comment:
can we re-use here the same structure used for `additional_endpoints` [here](https://github.com/DataDog/datadog-agent/blob/0a55a7fb2e233e19182f735223d556d500119a20/pkg/config/config.go#L309):

Reply:
Here `string` is used to define the name of the key in the yaml file, not the type of data stored in the key. I think that it should not be changed.